fixtures/american-dj/vizi-q-wash7.json

```
@@ -0,0 +1,849 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Vizi Q Wash7",
+  "shortName": "Q7",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Rey Victor", "Luke Nelson"],
+    "createDate": "2022-04-25",
+    "lastModifyDate": "2022-04-25"
```

How much mistakes can I make with updating the date? :sweat_smile: 

```suggestion
    "createDate": "2023-04-25",
    "lastModifyDate": "2023-04-25"
```

---

We're only human 🙂